Clarify test vectors fields. (#224)
cfrg/draft-irtf-cfrg-hpke: draft-irtf-cfrg-hpke.md

# Test Vectors

Each section below contains test vectors for a single HPKE ciphersuite and
contains the following values:

> 1. Input mode and private key material: This includes the `mode`, HPKE
>    ciphersuite identifiers (`kem_id`, `kdf_id`, `aead_id`), and all

Author: ```suggestion
1. Configuration information and private key material: This includes the `mode`, `info` string, HPKE
   ciphersuite identifiers (`kem_id`, `kdf_id`, `aead_id`), and all
```

>    sender, receipient, and ephemeral key material. For each role X,

Reviewer: ```suggestion
   sender, recipient, and ephemeral key material. For each role X,
```

>    where X is one of S, R, or E as sender, receiver, and ephemeral,

Reviewer: ```suggestion
   where X is one of S, R, or E as sender, recipient, and ephemeral,
```
